documents/installation/docker_installation.md

```diff
@@ -11,71 +14,84 @@ that isn't Ubuntu 22.04 LTS. It is a lighter-weight and faster alternative than
 
 [Ubuntu Instructions](https://docs.docker.com/engine/install/ubuntu/)
 
-## 2. Install VSCode (Highly Recommended)
+To check that everything installed OK, you should be able to open the command line and type:
+```bash
+docker
+```
+
+### INSTALLATION NOTES:
+* It's probably a good idea to restart your computer once you install Docker.
+* Docker has to be running before you can use it. When you install Docker, it usually enables itself upon the computer turning on. If this isn't the case, be sure to launch Docker before proceeding.
+* For Linux users: I recommed adding youself to the `docker` group. Being a member of the `docker` group allows you to run `docker` without `sudo`. After running the following two commands, log back out and in again for the changes to take affect.
```

Comment:
Try adding yourself to the `docker` group. I think this will fix issues later on.

Reply:
Just did, still don't understand why I wasn't automatically added though. Had to create the docker group from scratch too.  Maybe add a section on this in case anyone else gets the same error?

Reply:
Never mind, just missed the section you added about group creation